contracts/0.8.9/oracle/ValidatorsExitBusOracle.sol

```diff
@@ -159,18 +151,11 @@ contract ValidatorsExitBusOracle is BaseOracle, ValidatorsExitBus {
         bytes32 reportDataHash = keccak256(abi.encode(data));
         _checkConsensusData(data.refSlot, data.consensusVersion, reportDataHash);
         _startProcessing();
-        _handleConsensusReportData(data);
         _storeOracleExitRequestHash(dataHash, contractVersion);
+        _handleConsensusReportData(data);
         emit ExitDataProcessing(dataHash);
     }
 
-    /// @notice Returns the total number of validator exit requests ever processed
-    /// across all received reports.
-    ///
-    function getTotalRequestsProcessed() external view returns (uint256) {
```

Comment:
:thinking: @F4ever

Reply:
This was removed by mistake and returned:
https://github.com/lidofinance/core/blob/033f46bbc563732fd486cbc055ad741267616d92/contracts/0.8.9/oracle/ValidatorsExitBus.sol#L506

Reply:
Got it: it had been moved to VEB.sol (not VEBO.sol)